### ISSUE TITLE
texturs not loading

### ISSUE THREAD

**Issue by codechangerr** [ZippyCodeYT/Zippy_Codes#3]
i saved the textures to the correct path but it says no texture \car, enemy, road

**Comment by ZippyCodeYT** [comment]
Yes, it seems that there is a mismatch between code and project folder structure
can you show me your code?
or an image of your code and your project folder
